cypress/e2e/app_spec.cy.js

```diff
@@ -0,0 +1,46 @@
+describe('App', () => {
+  beforeEach(() => {
+    cy.intercept('GET', 'https://api.themotivate365.com/stoic-quote', {
+      statusCode: 200,
+      fixture: 'quote.json'
+    }).as('getQuote')
```

Comment:
I see you used an alias here but I do not see it in your code, you may not need this here.

Reply:
Yes thank you for catching this! I began to use it but decided not to and forgot to remove it.